### ISSUE TITLE
Error while running create_dataset_Original.py

### ISSUE THREAD

**Issue by Jayakrishnan-A** [nii-yamagishilab/Capsule-Forensics-v2#18]
Hai, I was trying to create original dataset (train) , and while executing create_dataset_Original.py I am getting read error . I have given path to FaceSwap mask folder, which are videos (--mask_ref).  Should I convert mask to images before executing? Please help me out. Thanks
![original_facswap_mask](https://user-images.githubusercontent.com/97498985/167268265-97e8aa9f-97a9-47a1-9c7c-f0abd02312a0.jpg)
![error](https://user-images.githubusercontent.com/97498985/167268271-2b3d5317-7b4b-47ce-83f4-a3c50475f3d7.jpg)

**Comment by honghuy127** [comment]
Hi,
Sorry for the trouble.
I updated the script. Could you please give it a try again?

**Comment by Jayakrishnan-A** [comment]
Sure I will try that. Currently i am trying to run the model inside a gpu enabled docker container **(pytorch/pytorch:1.3-cuda10.1-cudnn7-devel)** and getting the cuDNN error "**cuDNN error: CUDNN_STATUS_EXECUTION_FAILED**" while training for 1_deepfake dataset using [train_binary_ffpp.py]. Required PyTorch version is given as 1.3, may I know the used TorchVision, cuda  and cuDNN versions to check any dependency issues.  Thankyou.

**Comment by honghuy127** [comment]
Hi, here is the current version which works (Anaconda):
+ pytorch: 1.10.2
+ cudatoolkit: 11.1
 + cudnn: 8005

**Comment by Jayakrishnan-A** [comment]
> Hi, Sorry for the trouble. I updated the script. Could you please give it a try again?

Working perfectly. Thankyou.
